lib/http.js

```diff
@@ -138,7 +139,14 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
       if(!Array.isArray(invoker)) {
         invoker = [invoker];
       }
-      if(!invoker.includes(`https://${bedrock.config.kms.allowedHost}`)) {
+      let verified = false;
+      for(const [host] of bedrock.config.kms.allowedHosts) {
+        if(verified) {
+          break;
+        }
+        verified = invoker.includes(`https://${host}`);
```

Comment:
note to self, can invoker be modelled as a Map/Set

Reply:
it can be modeled as a `Set`. A few lines above this we do this:

```js
!Array.isArray(invoker) {
  invoker = [invoker];
} 
```

so you would need a new check:

```js
if(Array.isArray(invoker){
  invoker = new Set(invoker);
} else {
  invoker = new Set([invoker]);
}
```